ChangeLog.md

```diff
@@ -3,6 +3,308 @@
+researched.
+- Allowed wide/tall windows to spam multiple monitors
+- Added option to disable effect accounting in client, which could
+speed up GUI responses to things that would normally cause an
+accounting update, such as focus changes
+- Made spin controls on Options screen wider
+- Improved layout of various UI windows / popups for variable font sizes
+- Added "Update Design" and "Add New Design" commands to Design Screen
+- Added drop position indicator for completed designs list
+- Improved format and detail of combat log, including calculations for
+each attack and attackers being revealed by attacking
+- Added right-click Cut-Copy-Paste popup menus to text entry boxes
+- Made shift-delete cut and shift-insert paste in text entry boxes
+- Expanded set of allowed hotkey combinations
+- In SidePanel confine planet rename context menu item inside planet name
+- Moved Infrastructure indicator to population panel.
+- Moved supply meter from military to resources panel.
+- Options screen widgets to set objects list column widths
+- Tweaked how panels in fleets and planets lists resize with or
+without a scrollbar present
+- Added right-click popup menu command to copy text in most GUI text
+- Fixed issues where characters font textures weren't loaded even when
+using a suitable stringtable, making it impossible to use those
+characters in game
+- Added a right-click command to dismiss sensor ghosts on the fleets
+window, which causes an empire to forget a fleet existed, removing it
+from the map.
+- Disabled sound system init when sound and music are disabled
+- Fix crash when OpenAL device does not initialize
+- Added message box at startup informing player in case of audio
+system init failiure
+- Added message box at startup informing player in case of
+insufficient OpenGL version
+- Made pressing escape close more GUI windows
+- Added tech turn and RP requirements to tech panels on tech tree
+- Modified autogenerated fleet names to better reflect fleet contents
+- Increase fleet drop target width and add an icon to it
+- Modified how client and server communicate in a single player game
+to avoid firewall popups about network access
+- Added scanlines over fleets when not currently visible
+- Added nearly-accurate estimate of meter changes on the next turn to
+meter tooltips
+- Made MultiMeterStatusBar rescale to show largest value
+- In DesignWnd add ability to drag parts away from design
+
+
+#### Content
+
+- Translation updates: French, Russian
+- Removed / updated various missing, unused, or incorrect stringtable entries
+- Adjusted AI priorities for various buildings
+- Adjusted AI colonization planning
+- Made AI more cautious of hidden threats after losing fleets in a
+system it can't detect into
+- Made AI adjust priorities based on costs of ship parts and hulls
+- Adjusted AI priorities for research time
+- Adjusted AI industry modifiers
+- Adjusted AI weapon upgrade calculations
```

Comment:
Except for the losing-fleet part, imo the rest of the AI changes are small and unspecific enough to be summarized as "Various adjustments to AI priorities".